brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Timeline - Progress Bar not working in editor.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/0276c160-546a-484c-8c29-78f93f038788?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/ApuJbPb4

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the post timeline in the editor.
- Check the progress bar working fine in the editor after scroll. Change layout Orientation and check again.
- Check on responsive device Mobile/Tablet.

### Checklist:
